File: user-service/src/main/java/com/userservice/global/dto/ErrorResponse.java
package com.userservice.global.dto;

import lombok.Getter;

@Getter
public class ErrorResponse {
    private int status;
    private String errorMessage;

    private ErrorResponse(){}

    public ErrorResponse(int status, String errorMessage) {
        this.status = status;
        this.errorMessage = errorMessage;
    }
}

File: socat-community/post-service/src/main/kotlin/socat/postservice/global/dto/ErrorResponse.kt
package socat.postservice.global.dto

import java.time.LocalDateTime

class ErrorResponse(
        val timestamp: LocalDateTime,
        val status: Int,
        val error: String,
        val message: Map<String, String>?
) {
    constructor(status: Int, error: String) : this(LocalDateTime.now(), status, error, emptyMap())
    constructor(status: Int, error: String, message: Map<String, String>?) : this(LocalDateTime.now(), status, error, message)
}